amavisvt/client.py

```diff
@@ -518,15 +563,18 @@ def check_vt(self, checksums):
             send_checksums = sorted(list(set(raw_checksums + clean_hashes)))
             logger.debug("Sending %s checksums", len(send_checksums))
 
-            response = requests.post(self.config.api_url, {
+            #Call VT API for checksums           
+            response = self.api_call_check(send_checksums)
+            """response = requests.post(self.config.api_url, {
```

Comment:
Why is this block of code a multi-line string?

Reply:
Sorry, I'm very new to python. I thought it was fine for commenting.

Reply:
This way you're effectively creating a string which gets thrown away right away. The proper way to comment out code is with the hash sign:

```
# i'm a comment
```

However, if the code isn't used anymore, remove it.